### ISSUE TITLE
WebSocket object not receiving welcome message

### ISSUE THREAD

**Issue by alexemme** [semashkinvg/Bitmex.NET#49]
The object WebSocket is not receiving any welcome message, 
so it's not connecting to wss://www.bitmex.com/realtime,

does anyone know how to solve the issue?

**Comment by ridicoulous** [comment]
Yep, via my [library](https://github.com/ridicoulous/Bitmex.Net), feel free to use and contribute
